app/assets/stylesheets/alchemy/elements.scss

```diff
@@ -305,12 +305,12 @@ div#cells {
   right: 0;
 
   .icon_button {
+    @include button-defaults;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets